### ISSUE TITLE
Since accessibility fixes the IdP does not remember "selected device"

### ISSUE THREAD

**Issue by martin-lindstrom** [swedenconnect/bankid-saml-idp#175]
When the user first authenticates and then signs, the IdP should remember the last selected device so no question is asked during signing. In 1.0.1 (containing accessibility fixes from PR https://github.com/swedenconnect/bankid-saml-idp/pull/173) this doesn't work anymore.

**Comment by martin-lindstrom** [comment]
Fixed in PR https://github.com/swedenconnect/bankid-saml-idp/pull/176
